docs: add initial 1AD pages content (#27)
ariel-os/ariel-os.org: content/1ad/_index.md

draft = false
layout = 'community-day-2026'
type = 'page'
title = "The 1st Ariel OS Community Day (1AD)"
+++

## When? 

> September 1st 2026.

Reviewer: The date in the CfC page uses this format:
```suggestion
September 1, 2026.
```

## Where?
Grenoble, France.

## What is 1AD?

> 1AD, the 1st Ariel OS Community Day, is a meetup aiming to bring together embedded Rust developers, Ariel OS maintainers and Embassy contributors, beginners and experts, as well as people interested in the IoT in general and decision makers who consider deploying embedded Rust firmware/software in the near future. 1AD aims not only to inform about latest embedded Rust and Ariel OS developments but also to help gather feedback from the community so as to better shape the future of Ariel OS.

Reviewer: Trying to break up the first sentence:
```suggestion
1AD, the 1st Ariel OS Community Day, is a meetup aiming to bring together embedded Rust developers, Ariel OS maintainers and Embassy contributors, beginners and experts.
People interested in the IoT in general and decision makers who are considering deploying embedded Rust firmware/software in the near future are also very welcome!
1AD aims not only to inform about latest embedded Rust and Ariel OS developments but also to help gather feedback from the community so as to better shape the future of Ariel OS.
```

> The format of the event welcomes plenary talks, hands-on tutorials, and break-out sessions. We are currently calling for contributions. Go ahead and [propose a talk or a demo/tutorial](/1ad/call-for-contributions) for 1AD!

Reviewer: "Format of" sounds redundant to me:
```suggestion
The event welcomes plenary talks, hands-on tutorials, and break-out sessions. We are currently calling for contributions. Go ahead and [propose a talk or a demo/tutorial](./call-for-contributions) for 1AD!
```
